test/datastream/datastream_test.go

```diff
@@ -137,22 +137,17 @@ var _ = Describe("Streaming Data Conversion", func() {
 
 func getImageVirtualSize (outFile string) int64 {
 	//call qemu-img info
-	args1 := []string{"info", outFile}
 
 	//create command
-	//TODO: future investigate why this doesn't work
-	// 	cmd := fmt.Sprintf("qemu-img info %s | grep 'virtual size:'", outFile)
-	//  out, err := exec.Command("bash","-c",cmd).Output()
-	out, err := exec.Command("qemu-img", args1...).Output()
+	cmd := fmt.Sprintf("qemu-img info %s | grep 'virtual size:'", outFile)
+	out, err := exec.Command("sh","-c",cmd).Output()
```

Comment:
run `gofmt -w -stest/datastream/datastream_test.go` to format code per Go standards.

Reply:
ack